simulator/virtual_machine.go

```diff
@@ -1701,8 +1701,16 @@ func (vm *VirtualMachine) CloneVMTask(ctx *Context, req *types.CloneVM_Task) soa
 			pool = vm.ResourcePool
 		}
 	}
+
+	DestHost := vm.Runtime.Host
+
+	if req.Spec.Location.Host != nil {
+		hostRef := req.Spec.Location.Host.Reference()
```

Comment:
`req.Spec.Location.Host` is already a pointer, so there's no need for `hostRef` here, can do the following instead:
```go
	if req.Spec.Location.Host != nil {
		destHost = req.Spec.Location.Host
	}
```

Reply:
Sorry for that, I'm not extremely familiar with Go.
I amended my commit for that (since I also needed to change the commit message)